selfservice/flow/login/handler.go

```diff
@@ -348,8 +367,60 @@ type initializeSelfServiceLoginFlowForBrowsers struct {
 //	  400: jsonError
 //	  500: jsonError
 func (h *Handler) initBrowserFlow(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
-	a, err := h.NewLoginFlow(w, r, flow.TypeBrowser)
+	var hlr *hydraclientgo.LoginRequest
```

Comment:
Please add tests for this in handler_test.go

If you don't want to boot hydra, you could also use a `gomock` implementation of the hydra client